### DISCUSSION TITLE
Can't change the Global variable in the insight page

### DISCUSSION THREAD

**Discussion by on-future** [directus/directus#15201] [Archive]
![Screen Shot 2022-08-22 at 5 51 52 PM](https://user-images.githubusercontent.com/10277106/185880888-2cffb026-ea64-4bd0-ba8d-a9398703dba8.png)

The global variable is disabled in the show panel.

How can I edit it?

**Comment by azrikahar** [comment]
Seems like it is disabled due to #15050, particularly this commit https://github.com/directus/directus/pull/15050/commits/f944ed1dd8566ac2252246229b74b32aac916bf2, which commented that it prevents geometry type mismatch error. @rijkvanzanten just to confirm, global variable panels are meant to be editable on the fly right? If it is, then perhaps this disabling logic should (somehow) only be true for geometry type instead of disabling it for all 🤔 or maybe it should be something addressed downstream in the map interface, and not disabling it upstream.

**Reply by rijkvanzanten** [reply]
Ah yes that's the case. cc @joselcvarela those panels are editable on purpose, as they're global variables for use in other panels 👍🏻

**Reply by rijkvanzanten** [reply]
Ref https://github.com/directus/directus/pull/15211
